docs_source_files/content/webdriver/keyboard.en.md

```diff
@@ -1,193 +1,227 @@
+        await driver.get('https://www.google.com');
+
+        // Enter text "webdriver" and perform keyboard action "Enter"
+        await driver.findElement(By.name('q')).sendKeys('webdriver', Key.ENTER);
+    }
+    finally {
+        await driver.quit();
+    }
+})();
+  {{< / code-panel >}}
+  {{< code-panel language="kotlin" >}}
+import org.openqa.selenium.By
+import org.openqa.selenium.Keys
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.ChromeDriver
+
+class HelloSelenium {
+
+    fun main() {
+        driver = ChromeDriver()
+        try {
+            // Navigate to Url
+            driver.get("https://google.com")
+
+            // Enter text "q" and perform keyboard action "Enter"
+            driver.findElement(By.name("q")).sendKeys("q" + Keys.ENTER)
+        } finally {
+            driver.quit()
+        }
+    }
+}
+  {{< / code-panel >}}
+{{< / code-tab >}}
+
+## keyDown
+
+The keyDown is used to simulate action of pressing a modifier key(CONTROL, SHIFT, ALT)
+
+{{< code-tab >}}
+  {{< code-panel language="java" >}}
+WebDriver driver = new ChromeDriver();
+try {
+    // Navigate to Url
+    driver.get("https://google.com");
+
+    // Enter text "q" and perform keyboard action "Enter"
+    driver.findElement(By.name("q")).sendKeys("webdriver" + Keys.ENTER);
+
+    Actions actionProvider = new Actions(driver);
+    Action keydown = actionProvider.keyDown(Keys.CONTROL).sendKeys("a").build();
+    keydown.perform();
+} finally {
+    driver.quit();
+}
+  {{< / code-panel >}}
+  {{< code-panel language="python" >}}
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+driver = webdriver.Chrome()
+
+# Navigate to url
+driver.get("http://www.google.com")
+
+# Enter "webdriver" text and perform "ENTER" keyboard action
+driver.find_element_by_name("q").send_keys("webdriver"+Keys.ENTER)
+
+# Perform action ctrl + A (modifier CONTROL + Alphabet A) to select the page
+webdriver.ActionChains(driver).key_down(Keys.CONTROL).send_keys("a").perform()
+  {{< / code-panel >}}
+  {{< code-panel language="csharp" >}}
+IWebDriver driver = new ChromeDriver();
+try
+{
+    // Navigate to Url
+    driver.Navigate().GoToUrl("https://google.com");
+
+    // Enter text "q" and perform keyboard action "Enter"
+    driver.FindElement(By.Name("q")).SendKeys("webdriver" + Keys.Enter);
+
+    // Perform action ctrl + A (modifier CONTROL + Alphabet A) to select the page
+    Actions actionProvider = new Actions(driver);
+    IAction keydown = actionProvider.KeyDown(Keys.Control).SendKeys("a").Build();
+    keydown.Perform();
+}
+finally
+{
+    driver.Quit();
+}
+  {{< / code-panel >}}
+  {{< code-panel language="ruby" >}}
+require 'selenium-webdriver'
+driver = Selenium::WebDriver.for :chrome
+begin
+  # Navigate to URL
+  driver.get 'https://google.com'
+
+  # Enter text "cheese" and perform keyboard action "Enter"
+  driver.find_element(name: 'q').send_keys 'q', :return

```

Comment:
The code examples are not the same as the other ones, they are not sending `cheese`... Please double check when copying & pasting.
(Same applies to the Kotlin one)

Reply:
Sure @diemol , sorry for that. I will update the comments

Reply:
It is not only the comments, the actual code is not doing the same as the other examples.

Reply:
hi @diemol , the code sample is correct. only the comment is wrong. In all code samples it enters q for search and later it selects the page.

Reply:
and kotlin code just looks fine, i have tested multiple times. Can you please review again, i took hard time to test and replace the code didn't copy pasted :(

Reply:
Apologies, it just seems that overall the examples are sending different texts. I will merge and then fix it. 
Thanks!

Reply:
Sure, i will check the keyboard section again once site is updated. Sorry if i have done any mistake from my end too.

Thanks a lot @diemol.